src/pages/Main.js

```diff
@@ -1,26 +1,13 @@
-import { useState } from 'react'
+import 'src/libs/routes'
+
 import { h } from 'react-hyperscript-helpers'
-import { Ajax } from 'src/libs/ajax'
-import { useCancellation, useOnMount } from 'src/libs/react-utils'
-import { WorkflowSource } from 'src/pages/WorkflowSource'
+import { LocationProvider, Router } from 'src/libs/nav'
 
 
 const Main = () => {
-  const [cbasStatus, setCbasStatus] = useState()
```

Comment:
Yeah we can keep it. But we might need to talk with Tiffany to figure out the right UX for this. Maybe we can add it to bottom of the page instead of at top?

Reply:
This is mainly a debug/validation component while we build up the app and introduce error handling... So I don't think it's worth a design. My preference would be something like:
- Have a slightly ugly element until we're confident everything is working nicely
- Implement nice "oops, Cromwell/CBAS seems to be missing" error messages to catch the cases when things are broken
- Remove the ugly element